pootle/apps/contact/forms.py

```diff
@@ -18,10 +18,10 @@
 
 class ContactForm(MathCaptchaForm, OriginalContactForm):
 
-    field_order = ['name', 'email', 'subject', 'body', 'captcha_answer',
+    field_order = ['name', 'email', 'summary', 'body', 'captcha_answer',
```

Comment:
In other words, the patch to support Django 1.10+ requires setting the `subject()` method, which given the contact_form API effectively hides the `subject` field. In order to be able to use the `subject` field we need to rename to anything else (that doesn't clash with any reserved name in the contact_form API).